avr-specs/sync-from-upstream.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3
+
+import subprocess
+import json
+import os
+
+ENCODING = 'utf-8'
```

Comment:
I don't think explicitly defining the encoding should be necessary?  AFAIK Python already chooses a sane default.

Reply:
removed